src/bucket.js

```diff
@@ -177,10 +202,14 @@ export default class Bucket {
    * @return {Promise<Object, Error>}
    */
   setPermissions(permissions, options={}) {
-    return this.client.execute(requests.updateBucket({
-      id: this.name,
-      last_modified: options.last_modified
-    }, {...this._bucketOptions(options), permissions}));
+    if (typeof permissions !== "object") {
```

Comment:
Beware of this test for testing for actual objects, eg `null` and `array` are both passing this test (yeah, I know). 

Even if not blocking in this specific case, here's a simple helper you could probably use https://github.com/Kinto/kinto-admin/blob/97a61ec68a9608d48de4267072263f2fd24a37a6/scripts/utils.js#L21-L23